docs/source/overview.rst

```diff
@@ -45,6 +45,9 @@ following machine learning frameworks and packages:
 * :ref:`library-sklearn-crfsuite`. ELI5 allows to check weights of
   sklearn_crfsuite.CRF models.
 
+* :ref:`library-keras` - explain predictions of image classifiers
+  via Grad-CAM visualizations.
```

Comment:
Sorry for a late comment: could you please copy overview.rst changes to README file in the repo root?

Reply:
Will do that in https://github.com/TeamHG-Memex/eli5/pull/329.  Thanks